atom/renderer/atom_renderer_client.cc

```diff
@@ -70,19 +70,28 @@ void AtomRendererClient::DidCreateScriptContext(
     content::RenderFrame* render_frame) {
   RendererClientBase::DidCreateScriptContext(context, render_frame);
 
+  auto env = base::Environment::Create();
+  // NOTE: This should never be used in production, it is designed to be used
+  // for testing purposes
```

Comment:
Note that this is intentionally undocumented, or someone will come along and document it.